groups/chd-infrastructure/asg_frontend.tf

```diff
@@ -9,6 +9,30 @@ module "chd_fe_asg_security_group" {
   description = "Security group for the ${var.application} frontend asg"
   vpc_id      = data.aws_vpc.vpc.id
 
+  ingress_with_cidr_blocks = [
+    {
+      from_port   = 21
+      to_port     = 21
+      protocol    = "tcp"
+      description = "FTP Access"
+      cidr_blocks = join(",", local.admin_cidrs)
+    },
+    {
+      from_port   = 21
+      to_port     = 21
+      protocol    = "tcp"
+      description = "Allow FTP connections from internal NLB"
+      cidr_blocks = join(",", formatlist("%s/32", [for eni in data.aws_network_interface.nlb_fe_internal : eni.private_ip]))
+    },
+    {
+      from_port   = 21
+      to_port     = 21
+      protocol    = "tcp"
+      description = "Allow FTP connections from external NLB"
+      cidr_blocks = join(",", formatlist("%s/32", [for eni in data.aws_network_interface.nlb_fe_external : eni.private_ip]))
+    },
+  ]
+
```

Comment:
Will all client connections use active mode FTP? The absence of a port range other than the command port would suggest so here, but can we confirm that clients connections are not subject to traffic filtering and can accept inbound connections directly from the server on the client-specified data port?

Reply:
As discussed, further amendments have been made to define the ports used for passive mode FTP.
Updated in [288970b](https://github.com/companieshouse/chd-terraform/pull/18/commits/288970b4bd11f3d5455cbbd8cb45a8cac6f1977f) and [962d5a4](https://github.com/companieshouse/chd-terraform/pull/18/commits/962d5a4c5f4389e63638891e635f73cf91727ef8)